src/main/java/org/opensearch/knn/index/codec/derivedsource/DerivedMapHelper.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.index.codec.derivedsource;
```

Comment:
This is being tracked in https://github.com/opensearch-project/OpenSearch/pull/17612. Will remove this once that can be depended upon.